README.md

```diff
@@ -26,7 +26,7 @@ use autocxx::include_cpp;
 
 include_cpp!(
     #include "base/bob.h"
-    allow("Bob")
+    generate("Bob")
```

Comment:
```suggestion
    generate!("Bob")
```

<br>

Alternatively, making it work without the `!` as just `generate("Bob")` would be okay too.

Reply:
Ooh good spot, thank you!

Reply:
(and I've noticed there are other cases too)